### ISSUE TITLE
test_sqllite failure

### ISSUE THREAD

**Issue by tirkarthi** [kayak/pypika#265]
On running the project against Python 3.8 using pytest. On using `python setup.py test` the tests inside the dialect folder seem to be not executed. I ran the test under 3.5 and can still reproduce the test failure so it seems the tests were not executed in CI too catching this. 

```
 pytest
======================================== test session starts ========================================
platform linux -- Python 3.5.2, pytest-4.6.3, py-1.8.0, pluggy-0.12.0
rootdir: /home/karthi/pypika
collected 625 items

pypika/tests/test_aggregate.py ..................                                             [  2%]
pypika/tests/test_analytic_queries.py ...........................................             [  9%]
pypika/tests/test_criterions.py ............................................................. [ 19%]
....................................                                                          [ 25%]
pypika/tests/test_data_types.py .                                                             [ 25%]
pypika/tests/test_date_math.py ............................                                   [ 29%]
pypika/tests/test_deletes.py .....                                                            [ 30%]
pypika/tests/test_formats.py ..                                                               [ 31%]
pypika/tests/test_functions.py .............................................................. [ 40%]
........................................                                                      [ 47%]
pypika/tests/test_groupby_modifiers.py .............                                          [ 49%]
pypika/tests/test_immutability.py ..                                                          [ 49%]
pypika/tests/test_inserts.py ..............................................................   [ 59%]
pypika/tests/test_internals.py ....                                                           [ 60%]
pypika/tests/test_joins.py .....................................................              [ 68%]
pypika/tests/test_negation.py ....                                                            [ 69%]
pypika/tests/test_parameter.py ....                                                           [ 70%]
pypika/tests/test_pseudocolumns.py ........                                                   [ 71%]
pypika/tests/test_selects.py ................................................................ [ 81%]
.........................................................................                     [ 93%]
pypika/tests/test_tables.py ..........                                                        [ 94%]
pypika/tests/test_tuples.py .........                                                         [ 96%]
pypika/tests/test_updates.py ........                                                         [ 97%]
pypika/tests/test_utils.py ....                                                               [ 98%]
pypika/tests/test_vertica_queries.py ....                                                     [ 98%]
pypika/tests/dialects/test_mysql.py ....                                                      [ 99%]
pypika/tests/dialects/test_postgresql.py .                                                    [ 99%]
pypika/tests/dialects/test_sqllite.py FF                                                      [100%]

============================================= FAILURES ==============================================
________________________________ SelectTests.test_bool_false_as_zero ________________________________

self = <test_sqllite.SelectTests testMethod=test_bool_false_as_zero>

    def test_bool_false_as_zero(self):
        q = SQLLiteQuery.from_('abc').select(False)

>       self.assertEqual('SELECT 0 FROM "abc"', str(q))
E       AssertionError: 'SELECT 0 FROM "abc"' != 'SELECT false FROM "abc"'
E       - SELECT 0 FROM "abc"
E       ?        ^
E       + SELECT false FROM "abc"
E       ?        ^^^^^

pypika/tests/dialects/test_sqllite.py:18: AssertionError
_________________________________ SelectTests.test_bool_true_as_one _________________________________
import unittest


self = <test_sqllite.SelectTests testMethod=test_bool_true_as_one>

    def test_bool_true_as_one(self):
        q = SQLLiteQuery.from_('abc').select(True)

>       self.assertEqual('SELECT 1 FROM "abc"', str(q))
E       AssertionError: 'SELECT 1 FROM "abc"' != 'SELECT true FROM "abc"'
E       - SELECT 1 FROM "abc"
E       ?        ^
E       + SELECT true FROM "abc"
E       ?        ^^^^

pypika/tests/dialects/test_sqllite.py:13: AssertionError
```

Thanks

**Comment by twheys** [comment]
Hi, if you want to produce the output `SELECT 1` from pypika, I would suggest writing the python the same way. `SQLLiteQuery.from_('abc').select(True)`. Pypika does not try to be smart and translate all of the different versions of sql into their appropriate syntax, rather it let's you write python that translates 1:1 to SQL.

**Comment by tirkarthi** [comment]
The issue was more about the test being wrong and never ran on CI. Maybe the test can be fixed or the tests could be removed.

**Comment by twheys** [comment]
Ah, you're right. That's indeed a very different issue. Looks like the my CI isn't executing those tests and the code implementing that part is broken. Will push a fix.

**Comment by twheys** [comment]
Fixed in 0.28.1
